Library/Homebrew/cmd/update.sh

```diff
@@ -352,6 +373,11 @@ EOS
         exit
       fi
 
+      if [[ -n "$HOMEBREW_VERBOSE" ]]
+      then
+        echo "Fetching $DIR..."
+      fi
+      
```

Comment:
I'd rather leave this as-is; it was really useful when debugging as otherwise if a particular `git fetch` is happening/taking a long time it's really hard to figure out which one. Reminder this is for `--verbose` only. If you'd rather we can make it only for `HOMEBREW_DEBUG` but I personally want/need this output for debugging.